src/models/users.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { generateId } = require('../functions/generateId');
+
+class Expenses {
```

Comment:
Forgot to change the name here.

```suggestion
class Users {
```